bundles/org.openhab.binding.haassohnpelletoven/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="haassohnpelletoven"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Haas Sohn Thing Type -->
+	<thing-type id="oven">
+		<label>Haas and Sohn Pellet stove Binding Thing</label>
+		<description>The binding for haassohnpelletstove communicates with a Haas and Sohn Pelletoven through the optional
+			WLAN-Modul. More information can be found here: https://www.haassohn.com/de/ihr-plus/WLAN-Funktion</description>
+
+
+		<channels>
+			<channel id="channelIsTemp" typeId="isTemp"/>
+			<channel id="channelMode" typeId="mode"/>
+			<channel id="channelSpTemp" typeId="spTemp"/>
+			<channel id="channelPrg" typeId="prg"/>
+			<channel id="channelEcoMode" typeId="ecoMode"/>
+			<channel id="channelIgnitions" typeId="ignitions"/>
+			<channel id="channelMaintenanceIn" typeId="maintenanceIn"/>
+			<channel id="channelCleaningIn" typeId="cleaningIn"/>
+			<channel id="channelConsumption" typeId="consumption"/>
+			<channel id="channelOnTime" typeId="onTime"/>			
+		</channels>
+
+		<config-description>
+			<parameter name="hostIP" type="text" required="true">
+				<label>IP Address</label>
```

Comment:
You could specify `<context>network-address</context>` to get a free validation.

Reply:
keeping mine